frontend/packages/noobaa-storage-plugin/src/queries.ts

```diff
@@ -21,18 +36,33 @@ export enum StatusCardQueries {
   UNHEALTHY_POOLS = 'NooBaa_num_unhealthy_pools',
 }
 
-const PROJECT_CAPACITY_USAGE_QUERY = 'NooBaa_projects_capacity_usage';
-const BUCKET_CLASS_CAPACITY_USAGE_QUERY = 'NooBaa_bucket_class_capacity_usage';
-type ObjectCapacityQueryType = {
-  [queryType: string]: [string, string];
+export const CAPACITY_BREAKDOWN_QUERIES = {
+  [ObjectDashboardQuery.PROJECTS_BY_USED]: 'NooBaa_projects_capacity_usage',
+  [ObjectDashboardQuery.BUCKETS_TOTAL_USED]: 'NooBaa_bucket_class_capacity_usage',
 };
-export const ObjectCapacityQueries: ObjectCapacityQueryType = {
-  [ObjectDashboardQuery.CAPACITY_USAGE_PROJECT_QUERY]: [
-    `sort(topk(6, ${PROJECT_CAPACITY_USAGE_QUERY}{project!='Others'})) or ${PROJECT_CAPACITY_USAGE_QUERY}{project='Others'}`,
-    `sum(${PROJECT_CAPACITY_USAGE_QUERY})`,
-  ],
-  [ObjectDashboardQuery.CAPACITY_USAGE_BUCKET_CLASS_QUERY]: [
-    `sort(topk(6, ${BUCKET_CLASS_CAPACITY_USAGE_QUERY}{bucket_class!='Others'})) or ${BUCKET_CLASS_CAPACITY_USAGE_QUERY}{bucket_class='Others'}`,
-    `sum(${BUCKET_CLASS_CAPACITY_USAGE_QUERY})`,
-  ],
+
+export const breakdownQueryMap = {
+  [PROJECTS]: {
+    model: ProjectModel,
+    metric: 'project',
+    queries: {
+      [ObjectDashboardQuery.PROJECTS_BY_USED]: `sort_desc(topk(5, ${
+        CAPACITY_BREAKDOWN_QUERIES[ObjectDashboardQuery.PROJECTS_BY_USED]
+      }))`,
+      [ObjectDashboardQuery.PROJECTS_TOTAL_USED]: `sum(${
+        CAPACITY_BREAKDOWN_QUERIES[ObjectDashboardQuery.PROJECTS_BY_USED]
+      })`,
+    },
+  },
+  [BUCKETS]: {
```

Comment:
```suggestion
  [BUCKET_CLASS]: {
```
The metric is actually Bucket Class